src/App.js

```diff
@@ -0,0 +1,14 @@
+import React, { Route, Routes } from 'react-router-dom';
+import Home from './components/home';
+import Details from './components/details';
+
+const App = () => (
+  <div className="App">
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/details/:id" element={<Details />} />
+    </Routes>
+  </div>
```

Comment:
👍 Great work on your project! I noticed that you've successfully implemented the font family as mentioned in the previous review. However, there are a few areas that could use improvement to align more closely with the design:

- [ ] While you've used a beautiful red color shade throughout your code, I'd encourage you to follow the design guidelines and use the color specified in the design.
![image](https://github.com/AbuTalha3/react-csp-metric-app/assets/55840999/69ea00c9-4894-44cf-a856-cd46eb1dbcc5)

These updates will help ensure that your project closely matches the design and offers a more cohesive user experience. Keep up the great work! 👏🎨🌟